src/pevm.rs

```diff
@@ -328,6 +262,54 @@ pub fn execute_revm_sequential<S: Storage>(
     Ok(results)
 }
 
+fn canonize_account(
```

Comment:
`fully_evaluate_account` is a better name as it pairs well with lazy update (evaluation). `canonize` risks mistaking this with the concept of canonizing chains in consensus.